### DISCUSSION TITLE
expose PocketBase API/admin beyond localhost

### DISCUSSION THREAD

**Discussion by gapmiss** [pocketbase/pocketbase#321] [Show and tell]
I wrote up a tutorial on how I open up a local PocketBase API and admin to the Internet with a custom domain using a service called LocalXpose. LocalXpose is a reverse proxy that enables one to expose their localhost to the internet.

https://github.com/gapmiss/expose-database-server-pocketbase/blob/master/Expose%20database%20server%20(PocketBase).md

**Comment by ganigeorgiev** [comment]
Nice write up! For localhost tunneling I've used only https://ngrok.com/, but LocalXpose also seems like a nice tool.
